aiven/client/argx.py

```diff
@@ -105,6 +111,8 @@ def __init__(self, name):
         self.parser = argparse.ArgumentParser(prog=name, formatter_class=CustomFormatter)
         self.parser.add_argument("--config", help="config file location %(default)r",
                                  default=envdefault.AIVEN_CLIENT_CONFIG)
+        if __version__:
```

Comment:
Isn't this always true, either with real version or with "UNKNOWN" placeholder?

Reply:
Indeed.